contrib/google.golang.org/grpc/grpc.go

```diff
@@ -22,6 +22,21 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+func addCustomTags(cfg *config, opts ...tracer.StartSpanOption) []tracer.StartSpanOption {
```

Comment:
Maybe this is better suited as a method on `*config` inside `option.go`?
```go
// appendOpts creates a new set of StartSpanOptions by appending all options defined by configuration
// to opts.
func (cfg *config) appendOpts(opts ...tracer.StartSpanOption) []tracer.StartSpanOption {
```
You can probably include `tracer.WithAnalytics` into the body of this method because it seems to be added everywhere and is bound to config, unlike `tracer.Measured()` below. It will simplify things a bit.

Reply:
addressed